action/find.go

```diff
@@ -18,11 +19,28 @@ func (s *Action) Find(c *cli.Context) error {
 		return err
 	}
 	needle := strings.ToLower(c.Args().First())
+	choices := make([]string, 0, 10)
 	for _, value := range l {
 		if strings.Contains(strings.ToLower(value), needle) {
+			choices = append(choices, value)
+		}
+	}
+
+	if !s.isTerm {
+		for _, value := range choices {
 			fmt.Println(value)
 		}
+		return nil
 	}
 
-	return nil
+	act, sel := termwiz.GetSelection(choices)
+	fmt.Printf("User selected to %s %d\n", act, sel)
```

Comment:
Oh, yes indeed. This is left over debug code. Thanks for double checking. Will be gone in a sec.